src/test/kotlin/com/atlassian/performance/tools/virtualusers/measure/JiraNodeCounterTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.atlassian.performance.tools.virtualusers.measure
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.lang.StringBuilder
+
+class JiraNodeCounterTest {
+
+    @Test
+    fun shouldFailedIdentifyNotBreakTest() {
+        val jiraNodeCounter = JiraNodeCounter()
+
+        jiraNodeCounter.count(
+            BrokenApplicationNode()
+        )
+        val nodeIdentifyBuilder = StringBuilder()
+        jiraNodeCounter.dump(nodeIdentifyBuilder)
```

Comment:
The `dump` seems to be a part of post-conditions rather than the system under test

Reply:
done